iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -154,7 +154,7 @@ private void ConnectionStatusChangeHandler(ConnectionStatus status, ConnectionSt
                             {
                                 // Not great to print out a connection string, but this is done for sample/demo purposes.
```

Comment:
I don't think we should print out the connection string even if it is a sample and we have a statement that says it is not a good idea.

Reply:
I hear ya, I'll remove the connection string from the log statement.